desktop/apps/artwork/components/auction/helpers.coffee

```diff
@@ -1,12 +1,17 @@
 { compact } = require 'underscore'
+Auction = require '../../../../models/auction.coffee'
 
 pluralize = (word, count, irregular = null) ->
   if count is 1
     "#{count} #{word}"
   else
     "#{count} #{irregular or word + 's'}"
 
+liveAuctionUrl = (auction_id, user) ->
+  (new Auction(id: auction_id)).liveAuctionUrl(user)
```

Comment:
As this view is metaphysics-backed, it feels weird to mix in a Backbone model just for the sake of this helper. Makes me wonder if we move the `live_auction_url` logic to metaphysics for use in this view and the `my_active_bids` one.